### PR TITLE
Updating readme file IM-759

### PR DESCRIPTION
**Pull request by shaymark**
updating readme file
https://applicaster.atlassian.net/browse/IM-759

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+# Sport1 Player
+
+*Built by Applicaster*
+
+**Supports:** *iOS and Android*
+*Current Version: 0.0.21 (Android), 1.6.1 (iOS)*
+
+## About
+
+a player plugin for sport1, wrapper around JWPlayerPlugin
```
Comment (snooky23) [inline README.md:10]: Upper case A

File: README.md [hunk 2]
```diff
@@ -1 +1,49 @@
...
+
+### features and behaviour:
+
+- display the login plugin when needed (if playable is not free or if the user login token is missing for live content)
```
Comment (snooky23) [inline README.md:14]: An upper case for every first latter in the list

**Review by snooky23 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (shaymark) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (shaymark) [reply README.md]: fixed

**Pull request merged**